packages/sit-onyx/src/composables/useAnchorPositionPolyfill.ts

```diff
@@ -0,0 +1,85 @@
+import type { TooltipPosition } from "src/components/OnyxTooltip/types";
+import { ref, type ComputedRef, type ShallowRef } from "vue";
+import type { WedgePosition } from "./useWedgePosition";
+
+export const useAnchorPositionPolyfill = (
+  positionedRef: Readonly<ShallowRef<HTMLElement | null>>,
+  targetRef: Readonly<ShallowRef<HTMLElement | null>>,
+  positionArea: ComputedRef<TooltipPosition>,
+  alignment: ComputedRef<WedgePosition>,
+  alignsWithEdge: ComputedRef<boolean>,
+  fitParent: ComputedRef<boolean>,
+  offset: number,
+) => {
```

Comment:
(optional) to make the API more dev-friendly consider using an object as parameter and using `MaybeRefOrGetter`:

```suggestion
type UseAnchorPositionPolyfillOptions = {
  positionedRef: MaybeRefOrGetter<HTMLElement | null>;
  targetRef: MaybeRefOrGetter<HTMLElement | null>;
  positionArea: MaybeRefOrGetter<TooltipPosition>;
  alignment: MaybeRefOrGetter<WedgePosition>;
  alignWithEdge: MaybeRefOrGetter<boolean>;
  fitParent: MaybeRefOrGetter<boolean>;
  offset: MaybeRefOrGetter<number>;
};

export const useAnchorPositionPolyfill = ({
    positionedRef,
    targetRef,
    positionArea,
    alignment,
    alignWithEdge,
    fitParent,
    offset,
}: UseAnchorPositionPolyfillOptions) => {
```